### ISSUE TITLE
Possible To Set Gutter With To "0" In The BB

### ISSUE THREAD

**Issue by eduard-ungureanu** [elegantthemes/Divi-Beta#939]
### Problem:

Adding a new row into the layout, you can set the Custom Gutters to have values from 0 - 4. I'm not sure if this is a new layout addition, or the default values should be 1-3. When setting the custom gutters to be 4, the layout will be less smaller then using 3. 

After playing with this in VB and checking again the BB the Custom Gutters with can now be set only using values 1-4.

There is no difference between 0 and 1.

https://drive.google.com/open?id=0B8PjD3Z_XQvuaUJ3eURIUFU5QkU

## Attached PR

- https://github.com/elegantthemes/submodule-builder/pull/1743

**Comment by fikrirasyid** [comment]
@timayhelen I'm about to fix this but after checking it it appears that we intentionally allow 0 value for gutter width:

- related issue: https://github.com/elegantthemes/Divi/issues/1268
- related code: https://github.com/elegantthemes/submodule-builder/blame/master/main-structure-elements.php#L695

Based on our existing codebase, I think this should be a wontfix. Can you verify this?

**Comment by lots0logs** [comment]
@fikrirasyid The UI should reflect the fact that `1` is the lowest possible value. It didn't matter as much at the time of that old gh issue because the VB wasn't around then.

**Comment by fikrirasyid** [comment]
@lots0logs yeah, it is a legit point. I've submitted https://github.com/elegantthemes/submodule-builder/pull/1743 to fix this issue then 👍

**Comment by lots0logs** [comment]
@fikrirasyid Its still possible to save the value as zero. We should validate the input on the client side. Maybe setting `min` and `max` attributes on the `<input>` field would be enough :thinking:

**Comment by fikrirasyid** [comment]
@lots0logs I've incorporated your feedback on https://github.com/elegantthemes/submodule-builder/pull/1743/commits/54cce18be7b9903ac2d910c16a39af924b4455f3. Adding `min` and `max` attribute won't be enough since range input type is `text`. We can't use `number` because in same case we'd like to accept number unit as well. I've added js based validation to handle this on range input's keyup, change and settings modal submit.
